hcfb-be/backend-server/app/controller/distribution.controller.js

```diff
@@ -0,0 +1,75 @@
+const db = require('../config/db.config.js');
+const Distribution = db.distributions;
+
+exports.create = (req, res) => {
+    //Save to postgre database
+    Distribution.create({
+      "deliveryLocation" : req.body.deliveryLocation,
+      "deliveryDepartureTime" : req.body.deliveryDepartureTime,
+      "deliveryWeights" : req.body.deliveryWeights,
+      "projectedNumberOfClients" : req.body.projectedNumberOfClients
+    }).then(distribution => {
+        res.json(distribution);
+    }).catch(err => {
+        console.log(err);
+        res.status(500).json({msg: "error", details: err});
+    });
```

Comment:
We should also be able to extract this common `catch()` clause ☝️  to a reusable function. (Maybe in a follow-on PR.)